src/components/Base/TablePageSelector.tsx

```diff
@@ -41,6 +41,19 @@ function TablePageSelector(props: Props) : any {
   for (let i = numPagesLowerBound; i <= numPagesUpperBound; i += 1) {
     numPagesArray.push(i + 1);
   }
+  let pageDisplayNums : number[];
```

Comment:
can we convert some of these numbers to enums? there seems to be some arbitrary numbers (6, 3, 4, etc) so enums can help readers understand what these values actually represent

Reply:
I wasn't completely sure what some of that code did, but I can make some constants / try to see what they mean!